src/sinol_make/commands/run/__init__.py

```diff
@@ -706,6 +717,10 @@ def validate_arguments(self, args):
 
         return compilers, timetool_path
 
+    def exit(self):
+        if len(self.failed_compilations) > 0:
+            util.exit_with_error('Compilation failed for %d solution(s).' % len(self.failed_compilations))
```

Comment:
```suggestion
            util.exit_with_error('Compilation failed for {cnt} solution{letter}.'.format(cnt=len(self.failed_compilations), letter='' if self.failed_compilations.empty() else 's'))
```
(this suggestion might have a syntax error, this was hand-written)